src/app/[locale]/incident/add/components/IncidentQuestionsLocationForm.tsx

```diff
@@ -45,6 +54,18 @@ export const IncidentQuestionsLocationForm = () => {
     appendAdditionalQuestions()
   }, [formStoreState.main_category, formStoreState.sub_category])
 
+  useEffect(() => {
+    if (
+      formStoreState.coordinates &&
+      formStoreState.coordinates[0] === 0 &&
+      formStoreState.coordinates[1] === 0
+    ) {
```

Comment:
Misschien handig om een util te maken:

isCoordinates(arg: unknown): arg is [number, number] => ...